app/src/main/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenter.java

```diff
@@ -109,6 +109,12 @@ public void setView(NewsFeedContract.View view) {
         this.view = view;
     }
 
+    @Override
+    public void shareNews(int position) {
+        String messageToSend = news.get(position).getUrl();
+        navigator.sendPlainTextToApps(messageToSend);
```

Comment:
BTW, you're sharing a URL but not a text. Please, update the names of the corresponding methods for consistency.